.github/ISSUE_TEMPLATE/kubernetes-onboarding-template.md

```diff
@@ -1,57 +1,85 @@
 ---
-name: Kubernetes Onboarding Template
-about: For onboarding a new app to the VSP Kubernetes clusters
-title: '[Insert your app name here] - Onboarding to Kubernetes (EKS)'
-labels: 'operations, kubernetes, devops, eks, infrastructure, infrastructure-request'
-assignees: ''
+name: Application Onboarding Template
+about: For onboarding a software application to the Platform's delivery infrastructure (EKS, Kubernetes)
+title: '[Insert app name here] - Application onboarding infrastructure request'
+labels: 'operations, devops, eks, kubernetes, infrastructure, infrastructure-request'
+assignees: ''   
 
 ---
-## Description of your application
-_Please provide a brief description of the application, ie What does it do? Who does it serve?_
+## Summary
+The purpose of this template is to capture requirements for onboarding a software application to the Platform's delivery infrastructure. 
+
+### Overview
+- The Platform leverages container orchestration to host and deploy software applications  
+- Container orchestration automates the deployment, scaling, and management of containerized applications
+- To learn more, see ["Application Hosting and Deployment using Container Orchestration (EKS)"](https://vfs.atlassian.net/wiki/spaces/OT/pages/1474593866/Application+Hosting+and+Deployment+using+Container+Orchestration+EKS)
 
-### Required info: 
-Team Name:  
-Application Name:  
-Functionality:  
-Language/Stack:  
-Ports/Networking needed:  
-Other infrastructure needed e.g. Database, Redis, etc:  
-Anything else we need to know?:  
+### Guidance
+- Please fill out the sections below with as much info as possible 
+- If you don't have info or know the answer to a given prompt, _it's okay to leave it blank_
+- The Infrastructure Team will assist you with gathering requirements _and_ performing key setup and configuration tasks 
 
 ---
-## Background/Context/Resources
-_Please link to an initiative or other documentation that desribes the need and purpose of the application_
 
-## Technical Notes
-_Additional technical information that helps to explain the desired setup and configuration_
+## Description of application
+_Please provide a brief description of the application, ie What does it do? Who does it serve?_
```

Comment:
Your use of "ie" here is correct, but in the other instances below here where you use "ie", it should be "eg".

Reply:
Cool. I will remember that on the next iteration. :)